recipes/auditwheel/meta.yaml

```diff
@@ -0,0 +1,36 @@
+package:
+  name: auditwheel
+  version: "1.1.0"
+
+source:
+  fn: auditwheel-1.1.0.tar.gz
+  url: https://pypi.python.org/packages/source/a/auditwheel/auditwheel-1.1.0.tar.gz
+  md5: f42ec0ed1ce5e09e5ef5d3ce4f9826e6
+
+build:
+  noarch_python: True
```

Comment:
That's very true.  I've gone back and forth on this.  I really don't like conda's general rule of having to build out all packages when a single noarch package suffices (for example, for pure python packages).

If you try to install auditwheel with pip on WIndows, it succeeds.  Running it gives

    Error: This tool only supports Linux

My motivation was really avoiding separate Python builds for Linux.